Mathlib/Analysis/Calculus/FormalMultilinearSeries.lean

```diff
@@ -344,3 +348,54 @@ theorem constFormalMultilinearSeries_apply [NontriviallyNormedField 𝕜] [Norme
 #align const_formal_multilinear_series_apply constFormalMultilinearSeries_apply
 
 end Const
+
+section Linear
+
+variable [NontriviallyNormedField 𝕜]
+  [NormedAddCommGroup E] [NormedSpace 𝕜 E]
+  [NormedAddCommGroup F] [NormedSpace 𝕜 F]
+
+namespace ContinuousLinearMap
+
+/-- Formal power series of a continuous linear map `f : E →L[𝕜] F` at `x : E`:
+`f y = f x + f (y - x)`. -/
+def fpowerSeries (f : E →L[𝕜] F) (x : E) : FormalMultilinearSeries 𝕜 E F
+  | 0 => ContinuousMultilinearMap.curry0 𝕜 _ (f x)
+  | 1 => (continuousMultilinearCurryFin1 𝕜 E F).symm f
+  | _ => 0
+#align continuous_linear_map.fpower_series ContinuousLinearMap.fpowerSeries
+
+theorem fpower_series_apply_zero (f : E →L[𝕜] F) (x : E) :
+    f.fpowerSeries x 0 = ContinuousMultilinearMap.curry0 𝕜 _ (f x) :=
+  rfl
+
+theorem fpower_series_apply_one (f : E →L[𝕜] F) (x : E) :
+    f.fpowerSeries x 1 = (continuousMultilinearCurryFin1 𝕜 E F).symm f :=
+  rfl
+
+theorem fpowerSeries_apply_add_two (f : E →L[𝕜] F) (x : E) (n : ℕ) : f.fpowerSeries x (n + 2) = 0 :=
+  rfl
+#align continuous_linear_map.fpower_series_apply_add_two ContinuousLinearMap.fpowerSeries_apply_add_two
+
+attribute
+  [eqns fpower_series_apply_zero fpower_series_apply_one fpowerSeries_apply_add_two] fpowerSeries
+attribute [simp] fpowerSeries
+
+end ContinuousLinearMap
+
+end Linear
```

Comment:
This material is moved (verbatim) from `Analytic/Basic.lean` (it seems to belong better here since it is purely algebraic, without mentioning radius of convergence, and the other file is very very long).